### PR TITLE
Translation tracking Chinese

### PR DESCRIPTION
**Pull request by jasonqu**
There's been some updates on Getting Started guide, so here's the Chinese translations for the changes listed:

* 3a144d1...9a26dd0
* b15f916

as the response to #109 :)

### CODE REVIEW

**Comment by eed3si9n** [comment]
Did you miss sbt/website@3a144d1b3ffe7361361304db1e270445c1443fdf?

**Comment by jasonqu** [comment]
yes and fixed

**Comment by eed3si9n** [comment]
@pengisgood, @XuefengWu , @fsi206914 could one of you review this PR please?

File: src/tutorial/zh-cn/15-Full-Def.md [hunk 1]
```diff
@@ -6,62 +6,12 @@ out: Full-Def.html
   [More-About-Settings]: More-About-Settings.html
   [Using-Plugins]: Using-Plugins.html
 
-.scala 构建定义
+附录：.scala 构建定义
 -----------------------
 
-本小节假设你已经阅读了之前的章节，尤其是 [.sbt 构建定义][Basic-Def]和[更多关于设置][More-About-Settings]。
+本页面描述了`.scala`构建定义的旧的风格。在以前的版本的sbt中，`.scala`是用来创建多项目构建定义的唯一方式，但sbt 0.13 添加[多项目.sbt 生成定义][Basic-Def]，这也是推荐的样式。
```
Comment (fsi206914) [inline src/tutorial/zh-cn/15-Full-Def.md]: 但sbt 0.13 添加-> 但sbt 0.13 添加了

File: src/tutorial/zh-cn/15-Full-Def.md [hunk 2]
```diff
@@ -150,14 +100,6 @@ sbt 从 `.sbt` 文件*附加*设置到 `Build.settings` 和 `Project.setting` 
 - 在 `.sbt` 文件中的设置被 *追加* 到 `.scala` 中的设置。
 - 在 `.sbt` 文件中的设置是在项目作用域的，除非你指定它在其他域。
```
Comment (fsi206914) [inline src/tutorial/zh-cn/15-Full-Def.md]: 在项目作用域里，除非你指定它在其他域。

File: src/tutorial/zh-cn/12-Organizing-Build.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  [More-About-Settings]: More-About-Settings.html
+  [Using-Plugins]: Using-Plugins.html
+  [Library-Dependencies]: Library-Dependencies.html
+  [Multi-Project]: Multi-Project.html
+  [Plugins]: ../reference/Plugins.html
+
+组织构建
+--------------------
+
+本页将讨论构建结构的组织。
```
Comment (fsi206914) [inline src/tutorial/zh-cn/12-Organizing-Build.md]: 本页面

**Comment by jasonqu** [comment]
@fsi206914 thanks for review. If there is no other problem and the overall quality is good enough for publish, could you please approve this pr?:)

**Comment by fsi206914** [comment]
@jasonqu I'm kind of new. How to approve this?

**Comment by jasonqu** [comment]
leave a comment is OK, just let others know that you think this translation is ready

**Comment by eed3si9n** [comment]
Leave a comment "LGTM" (Looks good to me)

**Comment by eed3si9n** [comment]
@fsi206914 any updates?

**Comment by fsi206914** [comment]
Everything looks good to me.
LGTM.

**Comment by eed3si9n** [comment]
Merged and updated http://www.scala-sbt.org/0.13/tutorial/zh-cn/Installing-sbt-on-Linux.html.
Thanks guys.

**Comment by fsi206914** [comment]
Thanks :+1:

**Pull request merged**